meson.build

```diff
@@ -0,0 +1,82 @@
+project('exif', 'c', license: 'lgpl', version: '0.6.22')
+
+cc = meson.get_compiler('c')
+
+deps = [meson.get_compiler('c').find_library('m')]
```

Comment:
This should have `required: false` for portability (there is no libm on many platforms, it is part of libc).